database.rules.json

```diff
@@ -3,24 +3,57 @@
     "users": {
       "$uid": {
         ".read" : "auth !== null && auth.uid === $uid",
-        ".write" : "auth !== null && auth.uid === $uid"
+        "allCalendarIds" : {
+          //".write" : "true",
+          "$calendarId" : {
+            ".write" : "auth !== null && auth.uid === $uid && (!data.exists() || root.child('calendars/' + $calendarId + '/writers/' + $uid).exists())",
+            ".validate" : "newData.isString() && newData.val().length > 0"
+          }
+        },
+        "currentActiveCalendar" : {
+            ".write" : "auth !== null && auth.uid === $uid && (!data.exists() || root.child('calendars/' + newData.val() + '/writers/' + auth.uid).exists())",  
```

Comment:
Not important now, but if it's supposed to be possible to share a calendar somehow then we'll have to look at this again